docs/tutorials/average_optimizers_callback.ipynb

```diff
@@ -76,9 +76,9 @@
         "\n",
```

Comment:
<p>Please maintain spaces between <code>#</code> when you are commenting. I would also club together cells that are performing similar actions. For example,</p><p> </p><pre class="ql-syntax" spellcheck="false"># Build and train the model
...

# Load the latest checkpoints into the model
...
</pre><p> </p><p>Also, I am not sure why this line is needed now? <code>model.save_weights(vanilla_checkpoint_path.format(epoch=0))</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/2497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>About the spaces, sure. About format, I don't exactly know as to why it's done, I've used this as reference https://www.tensorflow.org/tutorials/keras/save_and_load. I'll try to remove and see it's effect, if its unnecessary I'll remove it :+1: </p><p> </p>